src/Core/Domain/Attachment/Command/AddAttachmentCommand.php

```diff
@@ -34,32 +34,32 @@ class AddAttachmentCommand
     /**
      * @var string|null
      */
-    private $pathName;
+    protected $pathName;
```

Comment:
Isn't a BC to change the visibility of a variable ?!

Reply:
Ping @PrestaShop/prestashop-core-developers

Reply:
It would be indeed if I make it private from protected (vice versa).
Now I doubt it.
BUT, I am not sure if its okay to extend the command with another one, maybe I should duplicate all the properties instead?
ping @matks, @jolelievre (you've already told your opinion in slack, but anyway :smile: )

Reply:
Regardless of the authorized extend or not, I don't think you need to change the visibility of these fields Since you can modify them via the public method setFileInformation anyway

Reply:
It's not a BC, it won't break anything.